### ISSUE TITLE
Option to use local katex

### ISSUE THREAD

**Issue by julianstirling** [mbarkhau/markdown-katex#6]
It would be great to have an option to pass a path to a local copy of katex so that we can create documentation that will work offline without needing to pull from https://cdn.jsdelivr.net/npm/katex@0.13.6/dist/katex.min.css

**Comment by mbarkhau** [comment]
Fair point. The url is currently hardcoded in [src/markdown_katex/html.py](https://github.com/mbarkhau/markdown-katex/blob/master/src/markdown_katex/html.py).

I would accept a PR to customize the styles that are currently hardcoded.

**Comment by mbarkhau** [comment]
I should note, if the idea here is to change the katex.min.css so that it is local, then you very likely want to also have all the font files referenced by it. I did something similar here a while ago: https://github.com/mbarkhau/litprog/blob/master/makefile#L102

**Comment by julianstirling** [comment]
Fantastic. Thanks for the speedy response. I will look into it when i get a chance.

**Comment by tylerebowers** [comment]
You can also just put the css you want to add at the bottom of the page, I've put mine in the footer.

```html
<style>
  .katex-display {
    /* override katex css */
    overflow-x: auto;
    overflow-y: hidden;
  }
</style>
```
